sample/kotlin/src/main/kotlin/com/datadog/android/sample/webview/WebFragment.kt

```diff
@@ -28,11 +28,15 @@ import androidx.lifecycle.ViewModelProviders
 import com.datadog.android.log.Logger
 import com.datadog.android.sample.BuildConfig
 import com.datadog.android.sample.R
+import io.opentracing.Scope
+import io.opentracing.util.GlobalTracer
 
 class WebFragment : Fragment() {
```

Comment:
It could be nice to have a span logging the page loading there, with the `onPageStarted()` and  `onPageFinished()` callbacks in the `WebViewClient`